demo

```diff
@@ -1,12 +1,30 @@
 #!/bin/bash
 
-
 CLIENT="-f docker-compose.client.yml"
 if [ -n "${NO_CLIENT}" ]; then
     unset CLIENT
     echo "-- disabling client containers"
 fi
 
+if [ "$#" -eq 0 ]; then
+    echo "Arguments passed to this command are passed directly to docker-compose, if you want to run the demo, run:"
+    echo ""
+    echo "'./demo up'"
+    exit 1
+fi
+
+# speed up first start by pulling containers in parallel
+if [[ "$(docker images | grep -q 'mendersoftware/deployments')" -eq 0 ]]; then
```

Comment:
`-eq 0` is not the right check, because `$()` will return empty string, not `0`. The correct thing is to drop the `[[` construct and call the command directly in the `if`.

Reply:
Oops, yes, my bad.